Extensions/AzureBlob/Cosmos.DataTransfer.AzureBlobStorage/AzureBlobSourceSettings.cs

```diff
@@ -17,4 +18,21 @@ public class AzureBlobSourceSettings : IDataExtensionSettings
     public string BlobName { get; set; } = null!;
 
     public int? ReadBufferSizeInKB { get; set; }
+
+    public bool UseRbacAuth { get; set; }
+
+    public bool EnableInteractiveCredentials { get; set; } = true;
```

Comment:
I'm not sure this makes sense for this to default as true

Reply:
Thanks for the feedback, I agree. I followed the same [pattern](https://github.com/AzureCosmosDB/data-migration-desktop-tool/blob/df9f09dce93491f60f2a7356867710bf1820d5af/Extensions/Cosmos/Cosmos.DataTransfer.CosmosExtension/CosmosSettingsBase.cs#L17C1-L18C1) of defaulting to true as the Cosmos extension. Should I remove the default from the Cosmos extension as well?

Reply:
I went ahead and removed true default from both the Azure Blob Storage extension and the Cosmos extension, figuring not at least it aligns with the documentation.